src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -502,6 +502,8 @@ export default {
           sku: this.classId,
           iscn: this.iscnId,
           isbn: this.iscnData?.contentMetadata?.isbn,
+          datePublished: this.iscnData?.recordTimestamp,
+          bookFormat: this.isNftBook ? 'https://schema.org/EBook' : undefined,
```

Comment:
```suggestion
          bookFormat: this.nftIsNFTBook ? 'https://schema.org/EBook' : undefined,
```

Reply:
Not sure if `isNftBook` should be undefined on the nftId page because it is usually passed as a prop using `nftIsNFTBook`